dist/components/progress-steps/_progress-steps.scss

```diff
@@ -0,0 +1,111 @@
+// Progress
+// ========
+//
+// Helps show users where they exist within a multiple-page workflow, such as a
+// checkout flow.
+
+
+// Configurable Variables
+// ---
+
+$progress-steps__item-bg-color: hsl(0, 0%, 95%) !default;
+$progress-steps__item-padding: if(variable-exists(v-space), $v-space, 10px) !default;
+$progress-steps__layer: if(variable-exists(stage-layer), $stage-layer, 10) !default;
+$progress-steps__current-bg-color: darken($progress-steps__item-bg-color, 10%) !default;
+$progress-steps__completed-bg-color: darken($progress-steps__item-bg-color, 20%) !default;
+
+// Arrow Divider Variables
+//
+// 1. These two variables together determine the angle of the arrow dividers
+
+$progress-steps__divider-angle: 90px !default; // 1
+$progress-steps__divider-width: 20px !default; // 1
+$progress-steps__divider-border-thickness: 2px !default;
+$progress-steps__divider-border-color: hsl(360, 100%, 100%) !default;
+
+
+.c-progress-steps {
+    display: flex;
+    overflow: hidden;
+}
+
+
+// Progress: Steps
+// ---
+//
+// Notes:
+// 1. For the absolutely positioned arrows
+// 2. Adjust padding to account for the arrows
+// 3. Coloured triangle
+// 4. Background color spacing
+// 5. Offset to create the appearance of a border
+
+.c-progress-steps__item {
+    position: relative; // 1
+
+    display: block;
+    padding: $progress-steps__item-padding;
+
+    background-color: $progress-steps__item-bg-color;
+
+    // 2
+    &:not(:first-child) {
+        padding-right: 0;
+        padding-left: $progress-steps__divider-width + $progress-steps__item-padding;
+    }
+
+
+    // Arrows
+
+    &::after,
+    &::before {
+        content: '';
+
+        position: absolute;
+        top: 50%;
+        left: 100%;
+        z-index: $progress-steps__layer;
+
+        display: block;
+        display: block;
+        width: 0;
+        height: 0;
+        border-top: $progress-steps__divider-angle solid transparent;
+        border-bottom: $progress-steps__divider-angle solid transparent;
+
+        transform: translateY(-50%);
+    }
+
+    // 3
+    &::after {
+        border-left: $progress-steps__divider-width solid $progress-steps__item-bg-color;
+    }
```

Comment:
Agree. Always prefer comments placed as close as possible to the source code. Footnote-style comments should only be used when adding lots of comments to a style block that would clutter the code. The referenced comment should be placed here instead of the 3.